scripts/clean_data.mjs

```diff
@@ -102,4 +108,4 @@ files.forEach((file) => {
 fs.writeFileSync(
   path.join(process.cwd(), "public/data/ovuzpsg_1221/cleaned", "all_data.json"),
   JSON.stringify(all_data, null, 2)
-);
\ No newline at end of file
+);
```

Comment:
my editor ran prettier on this file when i opened it- happy to revert but there's no changes here